src/Documentation/FixerDocumentGenerator.php

```diff
@@ -245,6 +245,18 @@ public function generateFixerDocumentation(FixerInterface $fixer): string
             {$fileName}
             RST;
 
+        $fileName = Preg::replace('~.*\K/src/(?=Fixer/)~', '/tests/', $fileName);
```

Comment:
This sets the path to the test file but keeps the name of the fixer class.

Reply:
One line belive the name is fixed too - is there any link that is not working?

Reply:
I meant the class name used as link label:
```diff
 Test class
 ------------
 
-`PhpCsFixer\\Fixer\\Alias\\ArrayPushFixer <./../../../tests/Fixer/Alias/ArrayPushFixerTest.php>`_
+`PhpCsFixer\\Tests\\Fixer\\Alias\\ArrayPushFixerTest <./../../../tests/Fixer/Alias/ArrayPushFixerTest.php>`_
```

Reply:
Looking at the docs output (e.g. `PhpCsFixer\\Fixer\\Alias\\ArrayPushFixer <./../../../tests/Fixer/Alias/ArrayPushFixerTest.php>`) it seems like FQCN is invalid, while file path is correct. But maybe it won't matter if we change to [this](#discussion_r1440572293).

Reply:
👍, fixed.